src/apps/search/panels/BasePanel.tsx

```diff
@@ -1,77 +1,312 @@
+  const { data: collection = {} }: { collection: Collection } = useLoader(onLoadManifests, null, [id]);
 
   /**
-   * Loads the first related media record from the Core Data API.
+   * Loads the related people from the Core Data API.
    */
-  const onLoadMedia = useCallback(() => Service.fetchRelatedMedia(id, { limit: 1 }), [id]);
-  const { data: { media_contents: mediaContents = [] } = {} } = useLoader(onLoadMedia);
+  const onLoadPeople = useCallback(() => Service.fetchRelatedPeople(id, { per_page: 0}), [id]);
+  const { data: { people = [] } = {} } = useLoader(onLoadPeople, null, [id]);
 
   /**
    * Loads the related place records from the Core Data API.
    */
-  const onLoadPlaces = useCallback(() => Service.fetchRelatedPlaces(id), [id]);
-  const { data: { places = [] } = {} } = useLoader(onLoadPlaces);
+  const onLoadPlaces = useCallback(() => Service.fetchRelatedPlaces(id, { per_page: 0 }), [id]);
+  const { data: { places = [] } = {} } = useLoader(onLoadPlaces, null, [id]);
+
+  /**
+   * Loads the related taxonomies from the Core Data API.
+   */
+  const onLoadTaxonomies = useCallback(() => Service.fetchRelatedTaxonomies(id, { per_page: 0 }), [id]);
+  const { data: { taxonomies = [] } = {} } = useLoader(onLoadTaxonomies, null, [id]);
+
+  /**
+   * Loads the related works from the Core Data API.
+   */
+  const onLoadWorks = useCallback(() => Service.fetchRelatedWorks(id, { per_page: 0 }), [id]);
+  const { data: { works = [] } = {} } = useLoader(onLoadWorks, null, [id]);
+
+  /**
+   * Memo-izes the bounding box options.
+   */
+  const bboxOptions = useMemo(() => getBoundingBoxOptions(config.map.max_zoom), [config]);
+
+  /**
+   * Memo-izes the base record.
+   */
+  const item = useMemo(() => data && data[props.name], [data, props.name]);
 
   /**
    * Memo-izes the geometry.
    */
-  const geometry = useMemo(() => !_.isEmpty(places) && CoreDataUtils.toFeatureCollection(places), [places]);
+  const geometry = useMemo(() => {
+    if (props.resolveGeometry && item) {
+      return props.resolveGeometry(item);
+    }
+
+    return !_.isEmpty(places) && CoreDataUtils.toFeatureCollection(places);
+  }, [item, places, props.resolveGeometry]);
 
   /**
-   * Memo-izes the image.
+   * Memo-izes the related media items.
    */
-  const image = useMemo(() => getPreviewImage(mediaContents), [mediaContents]);
+  const manifest = useMemo(() => {
+    if (_.isEmpty(collection.items)) {
+      return [];
+    }
+
+    const count = _.reduce(collection.items, (memo, item) => memo + item.item_count, 0);
+    const title = t('relatedMedia', { count });
+
+    return [{
+      items: _.map(collection.items, (item) => item),
+      renderItem: (item) => (
+        <ManifestThumbnail
+          className='py-2 px-6'
+          itemCount={item.item_count}
+          name={_.first(item.label?.en)}
+          onClick={() => setManifestUrl(item.id)}
+          thumbnail={item.thumbnail}
```

Comment:
One thing I noticed in Atlas is that if a thumbnail failed to generate or otherwise didn't exist, the related media items would become inaccessible. It might be worthwhile to do a conditional check for `item.thumbnail` and have a backup button that can still set the manifest url to the item id.

Reply:
👍 good call.

Reply:
Adding a `Button` component to render if no `thumbnail` prop is provided. We don't have a good media icon at this point, so I'm using the "info" icon as a placeholder.

![Screenshot 2025-02-20 at 6 22 16 AM](https://github.com/user-attachments/assets/e2d7f1db-ea53-46e8-91a6-4d49386e7825)

Reply:
Looks good!